src/renderer/scss/RecentConnections.scss

```diff
@@ -0,0 +1,52 @@
+h1 {
+  text-align: center;
+  margin: auto;
+}
+
+.RecentWrapper {
+  display: grid;
+
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.TDiv {
+  margin: auto;
+}
+.table {
+  text-align: center;
+}
+thead {
+  background-color: lightgrey;
+}
+
+th {
+  border: 5px solid;
+  text-align: center;
+  font-weight: bold;
+  font-size: large;
+}
+
+tr {
+  border: 5px solid;
+}
+
+td {
+  border: 3px solid;
+}
+.LinkTD {
+  border: none;
+}
+
+.buttonSelect {
+  border: none;
+  margin: 0;
+  background: none;
+}
+
+.buttonSelect:hover {
+  background-color: lightgreen;
+}
+
+.toHome {
+  margin: auto;
+}
```

Comment:
```suggestion
}
.HomeButton {
    margin: 5px;
    margin-right: 10vw;
}
```

Reply:
These two changes correspond with the home button alignment on the current execute page (Execute.scss) - thinking we do this for the sake of consistency. I can fix in a later PR so that the duplicate css classes are merged into the same file.

Reply:
Was able to get a styling similar to Execute using `justify-self: end;`

![Screen Shot 2022-09-26 at 8 54 59 PM](https://user-images.githubusercontent.com/112018180/192407184-3de3595c-f4cb-4c5e-aa7f-9f971ced1f27.png)